docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
-* isolation (default|hyperv)   (Windows daemon only)
+* isolation (process|hyperv)   (Windows daemon only)
```

Comment:
oh, hold on; I see you removed the 'default' option here already, could you put that back?

think I got fooled by GitHub caching here